djangocms_snippet/cms_plugins.py

```diff
@@ -39,8 +41,12 @@ def render(self, context, instance, placeholder):
             content = _('Template %(template)s does not exist.') % {
                 'template': instance.snippet.template}
         except Exception:
```

Comment:
I know this is legacy code but I think its best to catch this on save (clean method).
So we try to render the snippet on save and if it doesn't render then we provide a validation error.
I don't recommend trying to render the traceback because it adds more complexity to the plugin.

Reply:
OK, I do agree not to render the traceback. This was useful in order to understand code error triggerred when introducing menus in snippet, Cf. #35.

However, this code is wrong:
```
        except Exception:
            exc = sys.exc_info()[0]
            content = str(exc)
```
I see 2 alternatives:
- replacing 0 by 1: `exc = sys.exc_info()[1]`
- unify with https://github.com/divio/djangocms-snippet/blob/master/djangocms_snippet/templatetags/snippet_tags.py#L100:
```
        except Exception as e:
            content = str(e)
```

Reply:
I suggest to completely remove the exception handling from the rendering and instead move it to the form or model's clean method.

Reply:
Do you plan to do that in a short term horizon? If not, please can you merge this pull request as an interim solution?